integrations/zeroclaw/README.md

```
@@ -1,276 +1,189 @@
-# SEA-Forge ZeroClaw Integration
+# SEA-Forge ZeroClaw
 
-## Integration Flow
+### Run ZeroClaw with SEA authority enforcement before high-impact actions execute.
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)